### PR TITLE
 W-15946803 managing salesforce organizations sandbox feature jh

### PR DESCRIPTION
**Pull request by JennyHajee**

### CODE REVIEW

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+    <name>ContentAsset</name>
+</types>
+...
+----
+
+.. Review the metadata and retrieve it by running this command:
++
+[codeblock]
+----
+sf project retrieve start --source-dir force-app --target-org sandboxOrg
```
Comment (aamura) [inline modules/ROOT/pages/working-with-sandboxes.adoc]: Please, replace: 

sf project retrieve start --source-dir force-app --target-org sandboxOrg

by: 

sf project retrieve start --manifest ./config/package.xml --target-org sandboxOrg

Comment (JennyHajee) [reply modules/ROOT/pages/working-with-sandboxes.adoc]: fixed

**Review by aamura — DISMISSED** [review]

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 2]
```diff
@@ -0,0 +1,266 @@
+=  Working With Sandboxes
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Use Salesforce sandbox organizations to test your portal in a sandbox environment under the same conditions as your production environment. API Experience Hub enables you to manage separate deployment environments for your API portal. You can connect to multiple Salesforce sandbox organizations and switch between them. Manage Salesforce organizations from the *Manage your API portal* page by selecting sandbox environments that are available in the current Salesforce connection.
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]: Try to avoid "xxx enables you to". With AEH, you can...

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+Use Salesforce sandbox organizations to test your portal in a sandbox environment under the same conditions as your production environment. API Experience Hub enables you to manage separate deployment environments for your API portal. You can connect to multiple Salesforce sandbox organizations and switch between them. Manage Salesforce organizations from the *Manage your API portal* page by selecting sandbox environments that are available in the current Salesforce connection.  
+
+== Before You Begin
+Ensure that you have sandbox organizations already set up. For more information about creating sandboxes, see https://help.salesforce.com/s/articleView?id=sf.data_sandbox_create.htm&type=5[Create a Sandbox] in the Salesforce documentation.
+
+== Manage Salesforce Organization Connections
+
+. In the *Manage your API portal* page, click the connection control. 
++
+image::aeh-org-switcher.png["Connection control",15%]
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]: More context in the alt text? See https://confluence.internal.salesforce.com/display/MTDT/Updating+Alt+Text#UpdatingAltText-alt-text-guidelinesAltTextGuidelinesfromtheStyleReference

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+. To connect to a production organization, click *Connect production org* from the *Connected Salesforce Orgs* dialog. 
++
+The connection process restarts and enables you to connect to the production organization.
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]:
```suggestion
The connection process restarts and you can connect to the production organization.
```

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+. To switch to another a sandbox for the connected production organization, click *Switch to this org* from the *Connected Salesforce Orgs* dialog.
++
+The page displays the metadata for the sandbox organization.
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]: Necessary?

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+. To connect to a sandbox organization that isn't connected, click *Connect*.
++
+The connection process restarts and enables you to connect to the sandbox organization.
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]: I guess "enables you to" is important?

Comment (hannanelson) [reply modules/ROOT/pages/working-with-sandboxes.adoc]: If this is the same for each step, can you put it in the intro or end? Or use a partial?

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 7]
```diff
@@ -0,0 +1,266 @@
...
+
+== Before You Begin
+Ensure that you have sandbox organizations already set up. For more information about creating sandboxes, see https://help.salesforce.com/s/articleView?id=sf.data_sandbox_create.htm&type=5[Create a Sandbox] in the Salesforce documentation.
+
+== Manage Salesforce Organization Connections
+
+. In the *Manage your API portal* page, click the connection control. 
++
+image::aeh-org-switcher.png["Connection control",15%]
+. To connect to a production organization, click *Connect production org* from the *Connected Salesforce Orgs* dialog.
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]: Is this list sequential? It seems like they might be options, so bullets?

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 8]
```diff
@@ -0,0 +1,266 @@
...
+The page displays the metadata for the sandbox organization.
+. To connect to a sandbox organization that isn't connected, click *Connect*.
++
+The connection process restarts and enables you to connect to the sandbox organization.
+. To connect to a new sandbox organization, click *Connect new sandbox org*.
++
+The connection process restarts and enables you to connect to the new sandbox organization.
+. To switch to another sandbox organization, click *Switch to this org*.
++
+The page displays the metadata for the sandbox organization.
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]: Necessary?

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 9]
```diff
@@ -0,0 +1,266 @@
...
+
+== Manage Orphaned Sandboxes
+Orphan sandboxes occur when you disconnect a production organization and there are child sandbox organizations still connected. If you connect to a sandbox that isn't a child of the production organization, the sandbox appears in the *More sandbox orgs* section of the *Connected Salesforce Organizations* dialog. You can still switch to this sandbox. A sandbox no longer appear as an orphan when you reconnect to the associated production organization.
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]:
```suggestion
Orphan sandboxes occur when you disconnect a production organization when child sandbox organizations are still connected. If you connect to a sandbox that isn't a child of the production organization, the sandbox appears in the *More sandbox orgs* section of the *Connected Salesforce Organizations* dialog. You can still switch to this sandbox. A sandbox no longer appears as an orphan when you reconnect to the associated production organization.
```

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 10]
```diff
@@ -0,0 +1,266 @@
...
+
+== Manage Refreshed Sandbox Connections
+Refreshing a sandbox updates the metadata from the source organization. If a sandbox refreshes after the production organization is connected, reconnect to switch to this sandbox.
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc:35]: Can you make these active? Is the user doing the connecting?

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 11]
```diff
@@ -0,0 +1,266 @@
...
+
+If a sandbox refreshes before the production organization is connected, switch to this sandbox without.
+
+== Delete Sandbox Organizations 
+
+To delete a sandbox organizaton, see https://help.salesforce.com/s/articleView?id=000383260&type=1[Delete a sandbox] from the Salesforce documentation.
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]:
```suggestion
To delete a sandbox organizaton, see https://help.salesforce.com/s/articleView?id=000383260&type=1[Delete a sandbox] in the Salesforce documentation.
```

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 12]
```diff
@@ -0,0 +1,266 @@
...
+
+== Promoting a Sandbox to Production
+When a portal is tested in a sandbox environment and is ready for production, deploy a subset of the Salesforce metadata using your CI/CD pipeline to production. Afterward, connect to the production instance with the subset of the Salesforce metadata to complete the installation and start curating APIs.
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]: Passive

After testing a portal....?

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 13]
```diff
@@ -0,0 +1,266 @@
...
+
+//. Configure API Experience Hub in the sandbox organization:
+//.. Navigate to Anypoint Platform, enter your username and password, and then click *Sign in*.
+//.. From Anypoint Platform, select *API Experience Hub*.
+//.. Create a portal.
+//.. Add assets to the portal.
+//.. Preview your portal and add more customizations using Experience Builder.
+//.. Publish your portal and test it.
+
+Move all of the portal's styling and customizations from the sandbox organization to the production organization.
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]:
```suggestion
Move the portal's styling and customizations from the sandbox organization to the production organization:
```

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 14]
```diff
@@ -0,0 +1,266 @@
...
+   </types>
+   -->
+   <version>60.0</version>
+</Package>
+----
+====
+.. In a VCS, create a development branch.
+.. Download the metadata using the manifest created from the Salesforce sandbox organization. 
+. Retrieve the metadata from the Salesforce sandbox organization using Salesforce CLI commands:
+.. Log in to the Salesforce sandbox organization.
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]:
```suggestion
.. Log in to the Salesforce sandbox organization:
```

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 15]
```diff
@@ -0,0 +1,266 @@
...
++
+[codeblock]
+----
+sf auth web login --instance-url=https://<SANBOX-INSTANCE_URL> --alias sandboxOrg
+----
+.. Edit the `manifest/package.xml` file before retrieving the metadata from the sandbox organization to retrieve the new metadata that you added.
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]: Hard to parse. Read aloud.

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 16]
```diff
@@ -0,0 +1,266 @@
...
++
+Your sandbox organization can contain a large amount of metadata from different products because it's a copy of the Salesforce production organization. API Experience Hub is a subset of it and it's specified in the `manifest/package.xml` file that was created when creating the Git repository in a previous step.
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]: Multiple "it"/"it's". 

A subset of what? 
What is specified?

Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]:
```suggestion
Your sandbox organization can contain metadata from multiple products because it's a copy of the Salesforce production organization. API Experience Hub is a subset of the sandbox organization specified in the `manifest/package.xml` file created when you created the Git repository in a previous step. 
```

Comment (hannanelson) [reply modules/ROOT/pages/working-with-sandboxes.adoc]: (Not sure if that changes the meaning.)

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 17]
```diff
@@ -0,0 +1,266 @@
...
+
+.. Review the metadata and retrieve it by running this command:
++
+[codeblock]
+----
+sf project retrieve start --manifest ./manifest/package.xml --target-org sandboxOrg
+
+----       
+.. Confirm the metadata and commit all the metadata changes into the Git repository, 
+or you can modify the `manifest/package.xml` and repeat the process if some metadata is missing or not necessary.
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]:
```suggestion
or modify the `manifest/package.xml` and repeat the process if some metadata is missing or unnecessary.
```

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 18]
```diff
@@ -0,0 +1,266 @@
...
+. Move the API Experience Hub portal from the sandbox organization to the production organization:
+.. Log in to the production organization:
++
+[codeblock]
+----
+sf auth web login --instance-url=https://<ENTERPRISE-INSTANCE_URL> --alias productionOrg
+----
++
+Before installing the API Experience Hub Manage package in your production organization, get the managed package version ID (SubscriberPackageVersionId) that's installed in the sandbox organization.
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]: font for CamelCase bit?

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 19]
```diff
@@ -0,0 +1,266 @@
...
+.. Log in to the production organization:
++
+[codeblock]
+----
+sf auth web login --instance-url=https://<ENTERPRISE-INSTANCE_URL> --alias productionOrg
+----
++
+Before installing the API Experience Hub Manage package in your production organization, get the managed package version ID (SubscriberPackageVersionId) that's installed in the sandbox organization. 
++
+.. Run the following command to get the managed package version ID. The `SubscriberPackageVersionId` field from the API Experience Hub managed package begins with *04t*:
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]: mono here.

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 20]
```diff
@@ -0,0 +1,266 @@
...
+----
+sf package installed list --target-org sandboxOrg --json
+----
+.. Install the API Experience Hub managed package in the production organization:
++ 
+[codeblock]
+----
+sf package install --package 04t... --target-org productionOrg --no-prompt --wait 30
+----
+.. Deploy the API Experience Hub metadata to the production organization using Jenkins or other CI/CD tools to deploy the metadata into production, for example:
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]:
```suggestion
.. Deploy the API Experience Hub metadata to the production organization using Jenkins or other CI/CD tools, for example:
```

Comment (hannanelson) [reply modules/ROOT/pages/working-with-sandboxes.adoc]: (repetition)

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 21]
```diff
@@ -0,0 +1,266 @@
...
++
+[codeblock]
+----
+sf project deploy start --source-dir force-app --target-org productionOrg
+----
++
+All configurations deploy in the production organization and now you're ready to connect this organization with API Experience Hub.
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]:
```suggestion
All configurations deploy in the production organization and you're ready to connect this organization with API Experience Hub.
```

File: modules/ROOT/pages/working-with-sandboxes.adoc [hunk 22]
```diff
@@ -0,0 +1,266 @@
...
+. Configure API Experience Hub in the production organization:
+.. Navigate to Anypoint Platform, enter your username and password, and click *Sign in*.
+.. From Anypoint Platform, select *API Experience Hub* from the list of products.
+.. Connect the Salesforce production organization (Enterprise organization).
+.. Click *Use existing portal* to reuse the portal that already exists.
+.. From the *API Manage* page, add assets to the portal.
+.. Click *Preview and publish your portal* to preview the portal.
```
Comment (hannanelson) [inline modules/ROOT/pages/working-with-sandboxes.adoc]: combine with last step

File: modules/ROOT/pages/working-with-sandboxes.adoc (hunk 16 above)
Comment (JennyHajee) [reply modules/ROOT/pages/working-with-sandboxes.adoc]: Used your suggestion.

**Review by hannanelson — DISMISSED** [review]

**Review by dmerlob — APPROVED** [review]
